.github/workflows/chart-release.yml

```diff
@@ -30,6 +30,9 @@ jobs:
           echo "CHART_VERSION=$(yq e '.version' charts/provider-postgresql/Chart.yaml)" >> $GITHUB_ENV
           echo "PREVIOUS_CHART_VERSION=$(git show HEAD^:charts/provider-postgresql/Chart.yaml -m | grep "version:" | cut -d ' ' -f 2)" >> $GITHUB_ENV
 
+      - name: Prepare charts
```

Comment:
Is there a reason we are changing the chart release pipeline in this PR?

Reply:
When merging this PR to master I want to ensure that the CRD is up-to-date and attached to the next helm release. I'm not quite happy myself the way releasing helm charts conflicts a bit with `make generate`, I may come up with a PR that does things a bit better, but for now I say it's required.

Reply:
With #39 ff this should not longer be necessary.